code/modules/mob/living/carbon/human/species.dm

```diff
@@ -993,6 +998,11 @@
 			target.forcesay(hit_appends)
 		else if(target.lying)
 			target.forcesay(hit_appends)
+	if(target.martial_art && target.martial_art.block_chance)
```

Comment:
this is copy pasted everywhere, you need to reduce the copypaste by making it a method or some other means